src/plugins/data_views/common/field.stub.ts

```diff
@@ -23,6 +23,7 @@ export const stubFieldSpecMap: Record<string, FieldSpec> = {
   },
   'machine.os.raw': {
     name: 'machine.os.raw',
+    customLabel: 'OS',
```

Comment:
The controls unit tests use this field and so, by adding the custom label field here, we are able to test this functionality without any actual test changes.

Tested and this **does** cause the expected failure in the `control_editor` tests when we were improperly using `displayName`, and these tests pass again by switching to the field `name` instead.

Reply:
Hi @Heenawter,

What test is using this mock? It should be also possible to construct a data view with a custom field list with `createStubDataView`.

Reply:
We are using this mock in `src/plugins/controls/public/control_group/editor/control_editor.test.tsx` - we are using `createStubDataView` but we are using `stubFieldSpecMap` for our field list to avoid duplication. 

If you'd prefer to avoid changing this stub, I could spread `stubFieldSpecMap` instead and make the changes in our personal stub instead? Would that be better for ya'll? :)

Reply:
I think it would be better to define a custom one. It can still be based on `stubFieldSpecMap` plus your addition.

Reply:
Done in d5641f0a2303b4c9d8eaabb84167deb0644d31c8

Reply:
Thanks! LGTM 👍